src/SignalHandler.hpp

```diff
@@ -20,7 +20,7 @@
 
 #include "system.hpp"
 
-#include "signal.h"
+#include <csignal>
```

Comment:
I don't think this change is correct. `SignalHandler` uses `sigaddset` and friends which are documented to be part of `signal.h`, not `<csignal>`.